portfolio/src/main/webapp/script.js

```diff
@@ -6,15 +6,24 @@ function changeImage() {
 }
 
 /**
-  * Creates and returns an HTML list element containing
-  * a string provided by the the text parameter.
-  * @param {String} text Content of the list element.
-  * @return {<li>} HTML list element with the contents of the text parameter.
+  * Creates and adds a post card containing a caption and 
+  * a score as provided by the post parameter.
+  * @param {post} A user defined object with keys: (String)comment,
+  * (String)username, and (int)score. 
```

Comment:
The way you would do this is

``` @param {!{comment: string, username: string, score: number}}```

See [documentation](https://google.github.io/styleguide/jsguide.html#jsdoc-type-annotations).

You could also make post a class or ```@typedef```.